asterius/rts/rts.messages.mjs

```diff
@@ -0,0 +1,11 @@
+export class Messages {
+    constructor(memory) {
+        this.memory = memory;
+        Object.seal(this);
+    }
+
+    debugBelch2(fmt, arg) {
```

Comment:
iirc all `debugBelch2` calls are `debugBelch2("%s", c_str)`, so it makes sense to simply ignore the first format string here.

Reply:
True, but it doesn't _cost_ us to implement a general formatter, and we're correct to this spec, so I don't see a problem per se?